lib/archive_test.go

```diff
@@ -128,3 +128,5 @@ func TestArchiveJSONEscape(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Contains(t, string(b), "test<.js")
 }
+
+//TODO: write test for environment variables
```

Comment:
Yes please :)

Reply:
Sorry, looks like I missed a spot 😄. I think that I already wrote the `archive` test I intended to with this TODO as a part of the [end-to-end test here](https://github.com/na--/k6/blob/feature/set-env-vars-via-cli-2/cmd/runtime_options_test.go#L216-L240), so I'm not sure what exactly I can test that's not already covered by it. Any ideas or should I just delete the TODO?

Reply:
True that. Ok, just remove this TODO and we'll be ready to merge.

Reply:
Done